spec.emu

```diff
@@ -70,8 +70,8 @@ contributors: Google, Ecma International
       </p>
       <emu-alg>
         1. If _type_ is `"language"`, then
-          1. If _code_ does not matches the `unicode_language_id` production, throw a *RangeError* exceptipon.
-          1. If IsStructurallyValidLanguageTag(_code_) is *false*, throw a *RangeError* exception.
+          1. If _code_ does not matches the `unicode_language_id` production, throw a *RangeError* exception.  (This allows any Unicode CLDR locale identifier that does not contain extensions.)
```

Comment:
I think it is better to mention what it excludes, rather than what it allows by this step.